### DISCUSSION TITLE
Margins at bottom?

### DISCUSSION THREAD

**Discussion by directormac** [wezterm/wezterm#3855] [Q&A]
Good day contributors, i'd like to ask if there is an option to fill the window full? 
Even if i set the tab bar bottom or top there is still a gap on the bottom of the screen. 

are there any other options i have missed. Thank you in advance for response!

![image](https://github.com/wez/wezterm/assets/5866196/605c99f2-0117-44bb-b74e-ed254c6f03fd)

`config.window_padding = { left = 0, right = 0, top = 0, bottom = 0 }
config.window_frame = {
	border_left_width = "1px",
	border_right_width = "1px",
	border_bottom_height = "1px",
	border_top_height = "1px",
	border_left_color = colors.crust,
	border_right_color = colors.crust,
	border_bottom_color = colors.crust,
	border_top_color = colors.crust,
}
`

**Comment by wez** [comment]
See https://github.com/wez/wezterm/issues/3391#issuecomment-1492248423
